### ISSUE TITLE
Gems in Bank are not tracked

### ISSUE THREAD

**Issue by EntelligenteEnte** [Larsj02/Remix-Gem-Helper#8]
When I store surplus gems in the bank, their status goes from "In Bag" to "Uncollected".

**Comment by Larsj02** [comment]
yes as they're not in your bag and when you can't interact with them i don't really see a reason why they should be shown in the ui. also how do you put stuff in the bank are your bags not enough ?
